Automattic/wp-calypso
# [WordAds]: Update Privacy & Consent settings copy

#### Proposed Changes

* Update WordAds Privacy & Consent settings copy to reflect its applications to US states beyond California
For Context: New US states were added to the "Do Not Sell or Share My Personal Information" policy but the existing copy is specific to just California and this PR changes that in order to allow the copy to represent states beyond California.

#### Testing Instructions

- On a premium site
- Navigate to the admin page 
- In Tools -> Earn -> View ad dashboard -> Settings, scroll to the "Privacy and Consent"
- Activate the checkbox  "Enable targeted advertising to site visitors in all US states."
- You should see the new copy updates.

<img width="1048" alt="privacy and consent" src="https://user-images.githubusercontent.com/9039613/200368139-30ed46d3-b5eb-4b83-ac78-0602e50dc14a.png">
<img width="1040" alt="tooltip" src="https://user-images.githubusercontent.com/9039613/200368167-d046178f-e688-4119-80d3-bebf407ee503.png">

#### Pre-merge Checklist

 - [x] Make sure revision to rename shortcode from `[ccpa-do-not-sell-link]` to `[privacy-do-not-sell-link]` on WPCOM has been merged first
 - [ ] Confirm translation changes need for updated text





Related to #
